src/collector/mod.rs

```diff
@@ -386,6 +415,8 @@ impl Collector {
 
         // eprintln!("roots {:?}", roots);
 
+        // This step is dfs through the object graph (strating with the roots)
```

Comment:
Misspelled “starting” here.

Reply:
This is what I get for writing comments while tired... Will push fix tomorrow